src/compute/src/server.rs

```diff
@@ -217,6 +223,28 @@ impl mz_cluster::types::AsRunnableWorker<ComputeCommand, ComputeResponse> for Co
     }
 }
 
+/// Set the current thread's core affinity, based on the given `worker_id`.
+fn set_core_affinity(worker_id: usize) {
+    let Some(mut core_ids) = core_affinity::get_core_ids() else {
+        warn!(worker_id, "unable to get core IDs for setting affinity");
```

Comment:
Seems more like `error!` to me since this would mean we are taking a large (and unexpected) hit to performance.

Reply:
Seems fine. This is expected to return `None` on operating systems not supported by this crate, but I don't think we need to worry about those.